_episodes/04-Factory_Potential.md

```diff
@@ -204,14 +201,14 @@ Instead of using a set of if statements, we could use a Python dictionary to imp
 
 ~~~
 def potential_factory(potential_type, **kwargs):
-   cls_dict = dict(LJ=LJ, Buckingham=Buckingham)
+   cls_dict = {'LJ': 'LJ', 'Buckingham': 'Buckingham'}    # or `cls_dict = dict(LJ='LJ', Buckingham='Buckingham')`
```

Comment:
The idea here is to return an LJ or Buckingham object, so we do not need the quotes. As modified, `cos` on line 209 will be a string, and line 211 will not work. The original is correct here.

If this is not understandable in the lesson, we will need to modify the explanation.

Reply:
@janash Thank you for explaining it to me. It totally makes sense now that you said it. It just didn't click for me because I did not factor in that the `def potential_factory()` will be in the same file with the classes declaration. Thank you!